web-ui/src/app/components/tree-visualizer/tree-visualizer.js

```diff
@@ -348,7 +348,7 @@ var jQuery = require('jquery');
                 if ($this.is(":only-child")) {
                     if (li.data("lemma")) li.append("<span class='lemma'>" + li.data("lemma") + "</span>");
                     if (li.data("word")) {
-                        var csTitle = (args.showMatrixDetails && !li.data("caseinsensitive")) ? ' title="Case sensitive word"' : '';
+                        var csTitle = (args.showMatrixDetails && !li.data("caseinsensitive")) ? ' grBalloonPosition="down" grBalloon="Case sensitive word"' : '';
```

Comment:
This won't work.

Reply:
`'data-balloon-pos="down" data-balloon="Case sensitive word"'` should do the trick.